_episodes/05-comparing-agile-methodologies.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Comparing agile methodologies"
+teaching: 5
+exercises: 15
+questions:
+- "What are differences between Scrum and Kanban?"
+- "How do Scrum and Kanban relate to the Agile manifesto?"
+- "What are other Agile methodologies?"
+objectives:
+- "Explain the difference between Scrum and Kanban"
+- "Know that Scrum and Kanban are both Agile"
+- "Know that there are many other (less popular) methdologies"
+keypoints:
+- "Scrum and Kanban share similarities, but also have some important differences"
+- "Mix and match concepts from different methodologies to get to a hybrid methodology that matches your team best"
+---
+# Differences between Scrum and Kanban
```

Comment:
I created #14 to check consistency of this with episode 3 and 4 after they are done.